### PR TITLE
Add github pages support for highway

### PR DESCRIPTION
**Pull request by ax7e**
Add github pages support for hwy. 
@jan-wassenberg :
You need to change the following settings:
1. 
截屏2023-04-06 08 34 28
2.
截屏2023-04-06 08 42 11

### CODE REVIEW

File: .DS_Store
Comment (jan-wassenberg) [inline .DS_Store:1]: Let's remove this file? (Hidden, created by OS X) Also for docs/.

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+html_context['downloads'] = list()
+ 
+html_context['downloads'].append( ('epub', '/' +REPO_NAME+ '/' +current_language+ '/' +current_version+ '/' +REPO_NAME+ '_' +current_language+ '_' +current_version+ '.epub') )
+ 
+##########################
+# "EDIT ON GITHUB" LINKS #
+##########################
+ 
+html_context['display_github'] = True
+html_context['github_user'] = 'eric-lyun'
```
Comment (jan-wassenberg) [inline docs/conf.py:157]: What user should we have here?

**Review by jan-wassenberg — CHANGES_REQUESTED** [review]
Nice, thanks for adding this! Let's make some small changes before merging:

File: .DS_Store
Comment (ax7e) [reply .DS_Store:1]: Removed

File: docs/conf.py (hunk 1 above)
Comment (ax7e) [reply docs/conf.py:157]: This corresponds to the "Edit on GitHub" buttons on the website. I am in the process of fixing it.

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+import os
+import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'HIGHWAY'
+copyright = '2020, BSD-3-Clause License'
+author = 'Massimo DiPierro'
```
[the remark this answers is not in the document]
Comment (ax7e) [reply docs/conf.py]: I simply deleted it, it refers to the script only

**Pull request merged**